roles/edpm_nova/molecule/vagrant/molecule.yml

```diff
@@ -5,14 +5,16 @@ driver:
   name: vagrant
   provider:
     name: libvirt
+    type: libvirt
```

Comment:
i think this will set the default but i wold prefer to keep this isn the platform section is there a reason you moved this?

if you have multiple instance and you want to have different value then its prefer to define it in the platform section

Reply:
done, it was unintentional change